src/oracle.ts

```diff
@@ -17,6 +17,11 @@ export class OutdatedDataError extends Error {
     }
 }
 
+// Compares two serial numbers using RFC1982 serial number math.
+function serialNumberGt(i1: number, i2: number): boolean {
+    return (i1 < i2 && i2 - i1 > 2147483648) || (i1 > i2 && i1 - i2 < 2147483648);
```

Comment:
Please add a comment about 2147483648 being `2^31` and a link to https://tools.ietf.org/html/rfc1982

Reply:
I've changed it to use the hex constant to make it clearer.

Reply:
Did you push the change? I still see `return (i1 < i2 && i2 - i1 > 2147483648) || (i1 > i2 && i1 - i2 < 2147483648);`